qiskit_bot/notifications.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+import io
+import logging
+import multiprocessing
+import os
+import re
+
+import fasteners
+
+from qiskit_bot import git
+
+LOG = logging.getLogger(__name__)
+
+
+def trigger_notifications(pr_number, repo, conf):
+    """Do the post tag release processes."""
```

Comment:
Is this docstring accurate?

Reply:
Nope, it was sloppy copy and paste. It should be something like:
```suggestion
def trigger_notifications(pr_number, repo, conf):
    """Process any potential notifications on a new PR."""
```